repo/packages/B/bookkeeper/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+ "packagingVersion": "3.0",
+ "name": "bookkeeper",
+ "version": "4.3.4",
+ "scm": "https://github.com/twitter/bookkeeper",
+ "maintainer": "jia.zhai@emc.com",
+ "description": "bookkeeper",
+ "framework": false,
+ "tags": ["mesosphere", "service", "bookkeeper"],
+ "licenses": [
+   {
+     "name": "BSD",
+     "url": "http://www.apache.org/licenses/"
+   }
+ ],
+ "website":  "http://bookkeeper.apache.org",
+ "postInstallNotes": "bookkeeper on DCOS installed successfully!",
+ "preInstallNotes": "Be sure zk could access by master.mesos:2181, and bk will use port:3181",
```

Comment:
Yes, it has dependency with ZK. I thought it is always master.mesos:2181, so the docker image hard coded the port as: master.messo:2181